ramls/oaipmh/oaipmhupdatedinstanceids.json

```diff
@@ -0,0 +1,26 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Response schema for oai-pmh updated instances ids view",
+  "type": "object",
+  "properties": {
+    "instanceId": {
+      "description": "OAI-PMH updated instances ids",
+      "type": "string",
+      "$ref": "../uuid.json"
+    },
+    "instanceUpdatedDate": {
+      "description": "The last updated date or deleted date of an instance or it's items and holdings",
```

Comment:
The property name "instanceUpdatedDate" is misleading when the description says that it may be the updated date of an item.

Reply:
How do you think, will the name "oaiRecordUpdatedDate" be more appropriate for this field?

Reply:
Yes, "oaiRecordUpdatedDate" is fine, for me "updatedDate" is also sufficient because it is in the oaipmhupdatedinstanceids.json object, no need to repeat the object name in the object's properties.

Reply:
Thank you Julian, I'll use this name - "updatedDate".